lib/sample_rate_converter/Kconfig

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config SAMPLE_RATE_CONVERTER
+	bool "Sample Rate conversion library"
+	select CMSIS_DSP
+	select CMSIS_DSP_FILTERING
+	select TIMING_FUNCTIONS
+	help
+	  Enable the sample rate conversion library. The library uses CMSIS DSP filters to
+	  preserve quality during the conversion. 16k, 24k and 48k are supported.
+
+if SAMPLE_RATE_CONVERTER
+module = SAMPLE_RATE_CONVERTER
+module-str = Sample Rate Converter
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+config SAMPLE_RATE_CONVERTER_FILTER_SIMPLE
+	bool "Include the simple sample rate converter filters"
+	help
+	  Includes the simple filters for the sample rate converter. The simple filters uses the
+	  least amount of space and time for the conversion, at the expense of quality.
+
+config SAMPLE_RATE_CONVERTER_FILTER_SMALL
+	bool "Include the small sample rate converter filters"
+	help
+	  Includes the small filters for the sample rate converter. The small filters uses a small
+	  amount of space and time for the conversion, while also giving some low-pass filter
+	  capabilities.
+
+config SAMPLE_RATE_CONVERTER_MAX_FILTER_SIZE
+	int "Maximum number of filter taps the module supports"
+	default 34
+	help
+	  The maximum number of filter taps the sample rate converter supports. Increasing this
+	  number will increase the memory usage of the converter.
+
+config SAMPLE_RATE_CONVERTER_BLOCK_SIZE
+	int "Number of samples per conversion call"
+	default 480
```

Comment:
I thought this would be dependent on the sample rate input? In which case, we may want to rename to SAMPLE_RATE_CONVERTER_BLOCK_SIZE_MAX

Reply:
MAX is a better name, changing to that. It is indeed dependent on the input size, but as the text says the actual number of samples can be lower, so max is the correct name